sdk/nodejs/Makefile

```diff
@@ -29,12 +31,12 @@ build::
 	cp -R proto/. bin/proto/
 	mkdir -p bin/tests/runtime/langhost/cases/
 	find tests/runtime/langhost/cases/* -type d -exec cp -R {} bin/tests/runtime/langhost/cases/ \;
+	cd bin/ && \
+	cp ../yarn.lock . && \
+	(yarn unlink > /dev/null 2>&1 || true) && \
+	yarn link && \
+	yarn install --offline --production
```

Comment:
may be easier to read this if line continuations are indented

Reply:
Sure!